source/_docs/scripts.markdown

```diff
@@ -73,20 +73,20 @@ delay:
 ```yaml
 # Waits however many minutes input_number.minute_delay is set to
 # Valid formats include HH:MM and HH:MM:SS
-delay: {% raw %}'00:{{ states.input_number.minute_delay.state | int }}:00'{% endraw %}
+delay: {% raw %}'00:{{ states('input_number.minute_delay')|int }}:00'{% endraw %}
```

Comment:
* All examples containing Jinja2 templates should be wrapped **outside** of the code markdown with the `{% raw %}` tag.
* Use double quotes (`"`) for:
  * `friendly_name`
  * Single-line templates:
    * `value_template`
    * `level_template`
    * `icon_template`
    * Children of `data_template`